ci/test/00_setup_env_native_nowallet.sh

```diff
@@ -7,7 +7,7 @@
 export LC_ALL=C.UTF-8
 
 export CONTAINER_NAME=ci_native_nowallet
-export DOCKER_NAME_TAG=ubuntu:18.04  # Use bionic to have one config run the tests in python3.6, see doc/dependencies.md
```

Comment:
I think we could still leave this as 18.04, so we've got GCC and Clang builds on Bionic.

Reply:
Ah, makes sense, I missed that this one was covering bionic + clang as well. Fixed and amended the comment too.